tasks/src/image-segmentation/about.md

```diff
@@ -0,0 +1,38 @@
+## Use Cases
+The most important use of image segmentation is in computer vision for robotics, such as autonomous driving. Segmentation models are used to identify pedestrians, lanes and other necessary information. Image segmentation models are also used in cameras to erase background of portraits in images.
```

Comment:
I would avoid saying "the most important" since that's fairly subjective and open for discussion.